src/views.js

```diff
@@ -438,13 +438,23 @@ const AttachmentChatboxMessageView = new Lang.Class({
         let thumbnailFactory = Thumbnailer.forSize(GnomeDesktop.DesktopThumbnailSize.LARGE);
         let preview = getPreviewForFile(this.state.path, thumbnailFactory);
         if (preview.thumbnail) {
+            this.attachment_contents.orientation = Gtk.Orientation.VERTICAL;
+
             // Where we are using a thumbnail, use GtkCSSProivder to
             // set the background image
+            let naturalWidth = this.get_preferred_size()[1].width;
+            let minWidth = preview.thumbnail.width < naturalWidth ? naturalWidth : preview.thumbnail.width;
```

Comment:
Sorry to bikeshed on this... I think that this check will always end up with `preview.thumbnail.width`, because `this` could not possibly have a larger size, as it will only be sized afterwards (previously through `min-width`, now through the size request override).
What I would suggest is to simply keep the `min-width/min-height` setting as before, and adding `background-size: cover` (basically a one-liner). Does that work?